experimental/dds/tree2/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -662,9 +664,80 @@ describe("editable-tree: editing", () => {
 				field.setContent(undefined);
 			});
 
-			// delete method
+			// remove method
 			assert.throws(() => {
-				field.delete();
+				field.remove();
+			});
+		});
+	});
+
+	describe("treeStatus", () => {
+		describe("EditableTree", () => {
+			it("root node returns TreeStatus.InDocument", () => {
+				const view = createSharedTreeView().schematize({
+					schema: getTestSchema(FieldKinds.sequence),
+					allowedSchemaModifications: AllowedUpdateType.None,
+					initialTree: { foo: [], foo2: [] },
+				});
+				const rootNode = view.root;
+				assert(isEditableTree(rootNode));
+				const rootNodeStatus = rootNode[treeStatus]();
+
+				assert.equal(rootNodeStatus, TreeStatus.InDocument);
+			});
+
+			it("removed node returns TreeStatus.Deleted", () => {
```

Comment:
would be good to include a comment on or in this test for why it does not return "removed" yet.

Reply:
Added comment to test cases for clarification.